hail_scripts/v01/utils/add_hgmd.py

```diff
@@ -1,19 +1,46 @@
 import hail
 
+HGMD_PATH_GRCh37 = 'gs://seqr-reference-data-private/GRCh37/HGMD/HGMD_PRO_2018.1_hg19_without_DB_field.vds'
+HGMD_PATH_GRCh38 = 'gs://seqr-reference-data-private/GRCh38/HGMD/HGMD_PRO_2018.1_hg38_without_DB_field.vds'
+
+def reset_hgmd_fields_in_vds(hail_context, vds, genome_version, root="va.hgmd", subset=None, verbose=True):
+    hgmd_vds = read_hgmd_vds(hail_context, genome_version, subset=subset)
+
+    # %(root)s.review_status = NA:String,
+    vds = vds.annotate_variants_vds(hgmd_vds, expr="""
+        %(root)s.accession = NA:String,
+        %(root)s.class = NA:String
+    """ % locals())
```

Comment:
In general, using `locals` is really fragile because renaming variables can completely break things and its hard to know where if at all variables are being used. I know its all over this code base but I think we should try not to add any more

Reply:
This pattern seems equivalent to python 3.6  f-strings (https://www.python.org/dev/peps/pep-0498/), and it seems to me finding references to vars is ~ the same inside or outside these strings.

Reply:
I have fixed bugs caused by this multiple times in seqr. Maybe in theory its the same level of difficulty to find references to these variables but in practice this has broken on production